xmake.lua

```diff
@@ -42,7 +42,7 @@ target("cyber_engine_tweaks")
     set_kind("shared")
     set_filename("cyber_engine_tweaks.asi")
     add_files("src/**.cpp")
-    add_headerfiles("src/**.h", "build/CETVersion.h")
+    add_headerfiles("src/**.h", "src/**.hpp", "build/CETVersion.h")
```

Comment:
When you change extension, you can remove this change

Reply:
Removed ".hpp" references